app/assets/javascripts/headerFooter.js

```diff
@@ -6,6 +6,24 @@ jQuery(document).ready(function() {
 		jQuery("html, body").animate({scrollTop: 0}, duration);		 
 		return false;
 	})
+
+	let options = {
+		root: null,
+		threshold: .75
+	}
+	
+	function scrollToTop(entries, observer){
+		let scrollToTopBtn = $(".back-to-top");
+		entries.forEach(entry => {
+			if (entry.isIntersecting) {
+				scrollToTopBtn.fadeOut(300)
+			} else {
+				scrollToTopBtn.fadeIn(300)
+			}
+		})
+	}
+
+	let observer = new IntersectionObserver(scrollToTop, options);
+	observer.observe(document.querySelector("h1"));
```

Comment:
The problem is with this line. The observer uses a `h1` element to determine when to hide/show the `back-to-top` button and since the local env home page doesn't have a `h1` element the observer returns an error which makes the tests fail.

Reply:
Woohoo, nice! Good catch. Since you've tested this locally, I'd say this seems ready to merge!